src/unused_finder/unused_finder_visitor_runner.rs

```diff
@@ -114,7 +116,18 @@ pub fn get_import_export_paths_map(
         visit_module(&mut visitor, &resolved);
     });
 
+    let (leading, _) = comments.take_all();
+    let comments = leading.borrow();
+    let allow_unused = if let Some((_, comments)) = comments.iter().next() {
+        comments
+            .iter()
+            .any(|c| c.text.to_string().trim() == "@allow-unused-file")
```

Comment:
split c.text to lines and check each line in the comment -- otherwise you may end up fighting with various auto-formatters which may merge adjacent comments

In the current implementation, this will work:
```ts
// Implementation is underway, and will be consumed in future changes
// TODO: remove this allow-unused-file annotation after !18273 merges

// @allow-unused-file
```

but this will not:
```ts
// Implementation is underway, and will be consumed in future changes
// TODO: remove this allow-unused-file annotation after !18273 merges
// @allow-unused-file
```